homeassistant/components/alarmdecoder/config_flow.py

```diff
@@ -299,7 +299,7 @@ def _validate_zone_input(zone_input):
         errors["base"] = "relay_inclusive"
 
     # The following keys must be int
-    for key in [CONF_ZONE_NUMBER, CONF_ZONE_LOOP, CONF_RELAY_ADDR, CONF_RELAY_CHAN]:
+    for key in (CONF_ZONE_NUMBER, CONF_ZONE_LOOP, CONF_RELAY_ADDR, CONF_RELAY_CHAN):
```

Comment:
I'm surprised this isn't automatically handled by something like `pyupgrade`

Reply:
That would indeed be nice!

Reply:
That was my first though as well. Unfortunately it got rejected as not being in the "spirit of pyupgrade". Hence I had to implement it in pylint instead. If you like, feel free to leave a comment there: https://github.com/asottile/pyupgrade/issues/458